README.md

```diff
@@ -25,31 +25,31 @@ Each version of the plugin is published with a [specific set of versions of the
+| `addExtension`             | Whether to install the Datadog Lambda Extension as a layer. Defaults to `false`. When enabled, it's required to set the `apiKey` (or `apiKMSKey`) parameter. Learn more about the Lambda Extension Layer [here][8]. Note: AWS only supports Lambda Extensions for [certain runtimes][9].                                                                                                                                   |
+| `logLevel`                 | The log level, set to `DEBUG` for extended logging.                                                                                                                                                                                                                                                                                                                                                                        |
+| `enableXrayTracing`        | Set `true` to enable X-Ray tracing on the Lambda functions and API Gateway integrations. Defaults to `false`.                                                                                                                                                                                                                                                                                                              |
+| `enableDDTracing`          | Enable Datadog tracing on the Lambda function. Note: This applies only to integrations using the Datadog Extension. Defaults to `true`.                                                                                                                                                                                                                                                                                    |
+| `enableDDLogs`             | Enable Datadog log collection for the Lambda function. Note: This setting has no effect on logs sent via the Datadog Forwarder. Defaults to `true`.                                                                                                                                                                                                                                                                        |
+| `subscribeToAccessLogs`    | Enable automatic subscription of the Datadog Forwarder to API Gateway Access log groups. Requires that logging for the respective APIs are enabled in the provider section of the `serverless.yaml` file. Defaults to `true`. ]                                                                                                                                                                                            |
```

Comment:
Since we are removing the old parameters, this will be a breaking change and require updating the version number to 3.0.0. However, someone might still use the old parameters by accident. What happens if they do? Can we display some kind of helpful warning or error message?

Reply:
Any suggestion on how to do that? Did you mean in the README or in a log message?